libs/vertexai/langchain_google_vertexai/vectorstores/_searcher.py

```diff
@@ -176,3 +180,11 @@ def _get_deployed_index_id(self) -> str:
             f"deployed on endpoint "
             f"{self._endpoint.display_name}."
         )
+    
+
+    def _get_telemetry_user_agent(self) -> str:
```

Comment:
do we really need this as a separate function? it's used only once

Reply:
Maybe we don't wanted to be consistent with the other integrations, let me remove it and we can recrate it if we need it later on